pkg/packagekit/updates.less

```diff
@@ -51,6 +51,8 @@ tr.listing-ct-item {
         /* Mobile: force wrap version strings */
         td.version {
             word-break: break-all;
+            width: 16ex;
+            max-width: 30vw;
```

Comment:
Setting width and max-width for version in mobile mode lets us specify an optimum width to prevent wrapping a little more, but still provide breathing room when devices are extremely narrow.